xpdan/io.py

```diff
@@ -0,0 +1,41 @@
+try:
+    from skbeam.io.fit2d_io import fit2d_save, read_fit2d_msk
+except ImportError:
+    from skbeam.io.save_powder_output import _create_file_path
+    from fabio.fit2dmaskimage import Fit2dMaskImage
+    import fabio
+
+
+    def fit2d_save(mask, filename, dir_path=None):
+        """
```

Comment:
I am not 100% sure of PEP8 but generelly I like the docstring to have a one-liner (less than 80 characters) that is on the same line as the """, then a blank line, then a longer descpription if needed, then a blank line, then the input arguments, then a blank line, then what is returned.  I was taught that this works well with the auto-documenting features of python, though this may be out of date, so  just put me right in that case.

So as an example, this one might be something like this:
```
def fit2d_save(mask, filename, dir_path=None)
"""Takes a mask array and saves it in fit2D format

Arguments
------------
mask: ndarray
    The mask
filename: str
    The output mask-file filename
dir_path: str, optional
    Path to the destination file

Returns
---------
nothing
"""
```